extension/src/test/suite/experiments/index.test.ts

```diff
@@ -1527,85 +1522,49 @@ suite('Experiments Test Suite', () => {
 
   describe('Sorting', () => {
     it('should be able to sort', async () => {
-      const { dvcReader, internalCommands } = buildInternalCommands(disposable)
-
-      const messageSpy = spy(BaseWebview.prototype, 'show')
-
-      const resourceLocator = disposable.track(
-        new ResourceLocator(extensionUri)
-      )
-      stub(dvcReader, 'stageList').resolves('train')
-      stub(dvcReader, 'dag').resolves('')
-
-      const pipeline = buildExperimentsPipeline({
+      const { experiments, messageSpy } = await buildExperimentsWebview({
```

Comment:
[F] After reading this test I had no idea why it didn't use the builder so moved it across.